### PR TITLE
feat(core): bulk method for removing service from multiple resource

### PR DESCRIPTION
**Pull request by xflord**
* added bulk method to remove service from multiple resources in the same facility
* extended FacilityMismatchException to also include relevant facilities

### CODE REVIEW

File: perun-core/src/main/java/cz/metacentrum/perun/core/api/ResourcesManager.java [hunk 1]
```diff
@@ -534,7 +550,6 @@ public interface ResourcesManager {
 	 * @throws ServiceNotAssignedException
 	 */
 	void removeServices(PerunSession perunSession, Resource resource, List<Service> services) throws PrivilegeException, ResourceNotExistsException, ServiceNotExistsException, ServiceNotAssignedException;
-
```
Comment (HejdaJakub) [inline perun-core/src/main/java/cz/metacentrum/perun/core/api/ResourcesManager.java:537]: Mistakenly removed.

**Review by HejdaJakub — COMMENTED** [review]
It looks good otherwise.

**Review by Kuliak — COMMENTED** [review]
Good job!

File: perun-core/src/main/java/cz/metacentrum/perun/core/entry/TasksManagerEntry.java [hunk 2]
```diff
@@ -90,7 +90,7 @@ public void deleteTaskResultsByIds(PerunSession sess, List<Integer> taskResultId
 			Task currTask = tasksManagerBl.getTaskById(sess, currResult.getTaskId());
 			Facility currFac = currTask.getFacility();
 			if (currFac.getId() != facility.getId()) {
-				throw new FacilityMismatchException("Tasks are from different facilities.");
+				throw new FacilityMismatchException("Tasks are from different facilities.", facility, currFac);
```
Comment (Johaney-s) [inline perun-core/src/main/java/cz/metacentrum/perun/core/entry/TasksManagerEntry.java:93]: not sure this is privilege-wise okay, i think facility admin of another facility should not get whole facility object in the error message (as he's not privileged to call getFacilityById method)

File: perun-core/src/main/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntry.java [hunk 3]
```diff
@@ -590,6 +592,32 @@ public void removeService(PerunSession sess, Resource resource, Service service)
...
+		int facilityId = resources.get(0).getFacilityId();
+		for (Resource resource : resources) {
+			getResourcesManagerBl().checkResourceExists(perunSession, resource);
+			if (facilityId != resource.getFacilityId()) {
+				throw new FacilityMismatchException("Resources are from different facilities!", perunBl.getFacilitiesManagerBl().getFacilityById(perunSession, facilityId),
+					perunBl.getFacilitiesManagerBl().getFacilityById(perunSession, resource.getFacilityId()));
+			}
+
+			// Authorization
+			if (!AuthzResolver.authorizedInternal(perunSession, "removeService_Resource_Service_policy", Arrays.asList(resource, service))) {
```
Comment (Johaney-s) [inline perun-core/src/main/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntry.java:613]: authorization here should also go before throwing exception with facilities

**Review by Johaney-s — APPROVED** [review]

**Pull request merged**